Quiz/src/App.tsx

```diff
@@ -4,29 +4,38 @@ import Header from "./components/Header/Header";
 import Footer from "./components/Footer/Footer";
 import Start from "./components/Start/Start";
 import Quiz from "./components/Quiz/Quiz";
-
 import { QuizContext } from "./components/Helpers/Contexts";
 import Results from "./components/Results/Results";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Statistics from "./components/Statistics/Statistics";
 
 function App() {
   const GAME_STATE_MAP = {
     start: Start,
     quiz: Quiz,
     results: Results,
+    statistics: Statistics,
   };
   const [gameState, setGameState] =
     useState<keyof typeof GAME_STATE_MAP>("start");
   const Component = GAME_STATE_MAP[gameState];
   return (
-    <div className="wrapper">
-      <Header />
-      <main className="container">
-        <QuizContext.Provider value={{ gameState, setGameState }}>
-          <Component />
-        </QuizContext.Provider>
-      </main>
-      <Footer />
-    </div>
+    <BrowserRouter>
+      <div className="wrapper">
+        <Header />
+        <main className="container">
+          <QuizContext.Provider value={{ gameState, setGameState }}>
+            <Routes>
+              <Route path="/" element={<Component />} />
+              <Route path="/quiz" element={<Component />} />
+              <Route path="/results" element={<Component />} />
+              <Route path="/statistics" element={<Component />} />
```

Comment:
there is a potential problem
the thing is that `Component` depends on the `gameState` but routes doesn't
maybe would be better to bind components to their paths and then game state has to be the same as path
what do you think?